.gitignore

```diff
@@ -77,5 +77,10 @@ prime-router/source.mry
 prime-router/result_files/*
 prime-router/routed_files/*
 
+# Terraform files
```

Comment:
I don't think this `.gitignore` matches with this PR. Is this from your Terraform branch?

Reply:
I removed this.